tests/integration/test_integration.py

```diff
@@ -0,0 +1,79 @@
+# integration tests for micropip
+# these test often requires querying to the real packages existing in PyPI,
+# to test the micropip's ability to install packages from real world package indexes.
+
+# To prevent sending many requests to remote servers, these tests are disabled by default.
+# To run these tests locally, add `--integration` flag, when inovking pytest.
+
+from pytest_pyodide import run_in_pyodide
+
+
+def test_integration_install_basic(selenium_standalone_micropip, pytestconfig):
+    pytestconfig.getoption("--integration", skip=True)
```

Comment:
Can we make a separate fixture or decorator for this so that we can say:
```py
@integration_only
@run_in_pyodide
def test_integration_install_basic(selenium_standalone_micropip):
    # test body
```
Or maybe it would be possible to skip the entire file instead of each test individually?